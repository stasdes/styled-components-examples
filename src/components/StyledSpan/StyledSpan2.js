import styled from 'styled-components'
import styles from './StyledSpan2.styles'

export default styled.span.attrs({ draggable: true})`${styles}`





/* https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes */
