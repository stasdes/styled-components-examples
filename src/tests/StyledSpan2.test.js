import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import StyledSpan2 from '../components/StyledSpan/StyledSpan2'

test('StyledSpan2 should render correctly', () => {
  const tree = renderer.create(<StyledSpan2 />).toJSON()
  expect(tree).toMatchSnapshot()
})