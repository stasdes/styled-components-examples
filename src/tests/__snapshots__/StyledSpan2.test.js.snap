// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledSpan2 should render correctly 1`] = `
.c0 {
  color: green;
  font-size: 3rem;
}

.c0:hover {
  color: red;
}

<span
  className="c0"
  draggable={true}
/>
`;
